packages/lime-proto-bmx6/Makefile

```diff
@@ -7,7 +7,7 @@
 include $(TOPDIR)/rules.mk
 
 LIME_BUILDDATE:=$(shell date +%Y%m%d_%H%M)
-LIME_CODENAME:=bigbang
+LIME_CODENAME:=DaybootRelay
```

Comment:
this change has nothing to do with the topic of documentation, neither changes related with this package